lib/ansible/modules/cloud/vmware/vmware_deploy_ovf.py

```diff
@@ -311,6 +311,17 @@ def get_objects(self):
             cluster = find_cluster_by_name(self.si, self.params['cluster'])
             if cluster is None:
                 self.module.fail_json(msg="Unable to find cluster '%(cluster)s'" % self.params)
+            if self.params['resource_pool'] != 'Resources':
+                resource_pools = cluster.resourcePool.resourcePool
+                rp_exist = False
+                if len(resource_pools) != 0:
```

Comment:
This code should preferably go into the module_utils file and mirror the `find_resource_by_name` naming.  If another module needs this later, it will more difficult to relicense.

Reply:
Ok, will move to vmware.py, thanks.